_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -18,6 +18,8 @@ Users who have client systems that don't support TLS 1.2 should update those. RH
 
 EL5 does [not support TLS 1.2](https://access.redhat.com/articles/1294573) but is EOL.
 
+[The IETF official document](https://datatracker.ietf.org/doc/rfc8996/) contains more information on the deprecation of TLS 1.0 and 1.1.
```

Comment:
I think we should remove the upgrade warnings in nightly. This was just for 2.4 and should have been removed as part of branching. That way they contain only the warnings for 2.5.